lib/msal-core/src/Utils.ts

```diff
@@ -231,8 +231,8 @@ export class Utils {
    */
   static base64EncodeStringUrlSafe(input: string): string {
     // html5 should support atob function for decoding
-    if (window.btoa) {
-      return window.btoa(input);
+    if (Base64) {
```

Comment:
When is Base64 not defined?

Reply:
This should be true always, I added this to retain the existing code structure where we fall back to custom encode decode if needed. I can remove the fall back if you think we need to optimize it now. There is much of legacy code in Utils.ts which we can slowly move away from.

Reply:
to me it's less optimization and more removing confusion.
If its always true, lets remove that check

Reply:
So always is true is because we are importing it from the package.json; makes sense as this will standardize all our base64 encode and decode. Made the changes.

Reply:
Have added UT now. Please review.